### DISCUSSION TITLE
求助缓存转储的临时文件夹在那？

### DISCUSSION THREAD

**Discussion by YSYHZ** [hooke007/mpv_PlayKit#456] [Q&A 问答]
我用了这个脚本[streamsave](https://github.com/Sagnac/streamsave) （mpv 缓存转储命令的包装器）。
在观看bdmv时不小心按到输出文件的快捷键，bdmv太大了40个G，streamsave报错文件未写出。但是中间的临时文件未被清除
c盘直接满了。
现在想问一下临时文件夹在那里或者有什么有效的清理手段吗？
"C:\Users\yang\AppData\Roaming\mpv"这个路径已经找过了，没有
已经重启过两次了

**Comment by hooke007** [comment]
这种问题为什么不是第一时间找开发者本人。。。
没看脚本，就算不靠脚本也可以靠各种磁盘内容分析工具定位到大容量文件位置

**Reply by YSYHZ** [reply]
用了好几种分析工具了都没找到，我英语不太好，开发者是外国人，我去试试吧

**Reply by hooke007** [reply]
你首先临时文件夹清理了吗？
